event-sourcing/event-subscription-registry/src/main/java/uk/gov/justice/subscription/registry/SubscriptionsDescriptorsRegistry.java

```diff
@@ -38,22 +38,22 @@ public Subscription getSubscriptionFor(final String subscriptionName) {
      * @return subscription component name
      */
     public String findComponentNameBy(final String subscriptionName) {
-        final SubscriptionDescriptorDefinition first = registry
+        final SubscriptionsDescriptor first = registry
                 .stream()
                 .filter(subscriptionDescriptorDefinition -> isSubscriptionNameExist(subscriptionName, subscriptionDescriptorDefinition))
                 .findFirst()
                 .orElseThrow(() -> new RegistryException(format("Failed to find service component name in registry for subscription '%s' ", subscriptionName)));
         return first.getServiceComponent();
     }
 
-    private boolean isSubscriptionNameExist(final String subscriptionName, final SubscriptionDescriptorDefinition subscriptionDescriptorDefinition) {
-        return subscriptionDescriptorDefinition
+    private boolean isSubscriptionNameExist(final String subscriptionName, final SubscriptionsDescriptor subscriptionsDescriptor) {
+        return subscriptionsDescriptor
                 .getSubscriptions()
                 .stream()
                 .anyMatch(subscription1 -> subscription1.getName().equals(subscriptionName));
```

Comment:
Maybe rename subscription1 to just plain subscription?

Reply:
renamed